### ISSUE TITLE
Fix MyGet deployment permissions

### ISSUE THREAD

**Issue by MikeGoldsmith** [OrleansContrib/OrleansCouchbaseProvider#13]
The deployment permissions for pushing to MyGet are invalid, this is because the OrleansContrib account on AppVeyor will hash the password differently to my personal account.

@richorama will likely need your help to setup the AppVeyor secure hashes.

**Comment by ashkan-saeedi-mazdeh** [comment]
@MikeGoldsmith  , I can not help but @richorama  can I guess. He sets up CI for the repos as far as I know.

**Comment by richorama** [comment]
hmm, I'm not sure I have the key for MyGet, asking on gitter...
Which projects currently push to MyGet? I think mine just go to NuGet.

**Comment by MikeGoldsmith** [comment]
@richorama I've setup the CouchBaseProviders project to push to MyGet. I also created a test feed but can change to a different feed if there is a more appropriate one.

What we need to do:
1. Encrypt a MyGet key that has access (either mine or someone else) using @richorama's AppVeyor account. This is because each AppVeyor account has a different encryption key and that account is being used to run the builds for this repo.
2. Update appveyor.yml with encrypted key to MyGet
3. Update appveyor.yml MyGet feed URL if a different feed is more appropriate

To be fair, we don't need to integrate with MyGet. I just find it useful for a faster review process to maintain an unofficial MyGet feed that each commit pushes to. Happy to remove and just push to Nuget if you prefer.

**Comment by richorama** [comment]
@MikeGoldsmith  thanks for the explanation!

Happy to encrypt a key if someone is able to create one.

I notice that AppVeyor is also able to publish a nuget feed, I wonder if this provides the same feed you'd get with MyGet?

**Comment by MikeGoldsmith** [comment]
Yeah, I can send you my key for you to encrypt or you can create a MyGet account and I'll add you as a contributor to the current feed.

The project has been configured to deploy to both MyGet and Nuget. MyGet for every successful commit / build and to Nuget when the master branch is tagged. It's the same configuration we use for the official Couchbase SDK which works quite well.

**Comment by ashkan-saeedi-mazdeh** [comment]
Merged the PR, Can we close this guys? @MikeGoldsmith  @richorama ?
